Mathlib/LinearAlgebra/Dimension/Finrank.lean

```diff
@@ -38,7 +38,7 @@ open Cardinal Submodule Module Function
 variable {R : Type u} {M : Type v} {N : Type w}
 variable [Ring R] [AddCommGroup M] [Module R M] [AddCommGroup N] [Module R N]
 
-namespace FiniteDimensional
+namespace Module
```

Comment:
I think that at least `FiniteDimensional.finrank` should survive as a deprecated alias.

Reply:
Added